qbft/spectest/tests/roundchange/valid_justification_late.go

```diff
@@ -0,0 +1,42 @@
+package roundchange
+
+import (
+	"github.com/bloxapp/ssv-spec/qbft"
+	"github.com/bloxapp/ssv-spec/qbft/spectest/tests"
+	"github.com/bloxapp/ssv-spec/types"
+	"github.com/bloxapp/ssv-spec/types/testingutils"
+)
+
+// ValidJustificationLate tests a valid rc quorum justification
```

Comment:
explanation is not complete, please write better detail

Reply:
:+1: